contrib/build_static_lib.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -eE
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+# This scripts creates a single static UCX library, which also includes
+# ibverbs and ibcm libs.
+
+usage() {
+    echo "Usage: $0 [option...]"
+    echo "Builds a single static UCX library with ibverbs and ibcm included"
+    echo
+    echo "   -u, --ucx <PATH>          UCX installation prefix"
+    echo "   -i, --ib  <PATH>          infiniband verbs installation prefix (/usr)"
+    echo
+    exit 1
+}
+
+while [ "$1" != "" ]; do
+key="$1"
+case $key in
+    -u|--ucx)
+    [ -z $2 ] && usage
+    UCX_PREFIX="$2"
+    ;;
+    -i|--ib)
+    [ -z $2 ] && usage
+    IB_PREFIX="$2"
+    ;;
+    *)
+    usage
+    ;;
+esac
+shift 2
+done
+
+# UCX prefix is required, IB prefix is /usr by default
+[ -z $UCX_PREFIX ] && usage
+IB_PREFIX="${IB_PREFIX:-/usr}"
+
+[ -d "${IB_PREFIX}/lib64" ] && ib_lib_suffix="64"
+
+UCX_LIB_PATH="${UCX_PREFIX}/lib"
+IB_LIB_PATH="${IB_PREFIX}/lib${ib_lib_suffix}"
```

Comment:
maybe we need IB_LIB_OUT_PATH where to store libucx.a and cmd line to override it